README.md

```diff
@@ -6,168 +6,208 @@ level: 100
 service: ASP.NET Core Web App
 endpoint: AAD V2
 ---
-# Integrating Azure AD V2 into an ASP.NET Core web app
+# Integrating Azure AD V2 into an ASP.NET Core web app that calls the Microsoft Graph
```

Comment:
perhaps 'and call the Microsoft Graph API on behalf of a user'?

Reply:
Made this change.